docs/diagrams/README.md

```diff
@@ -0,0 +1,18 @@
+# Diagrams
+
+We are using [C4-PlantUML](https://github.com/plantuml-stdlib/C4-PlantUML) to document the architecture and design of Tock.  C4-PlantUML combines the benefits of PlantUML and the C4 model for providing a simple way of describing and communicate software architectures.  [PlantUML](http://en.plantuml.com/) is an open source project that allows you to create UML diagrams using code.  [C4 model](https://c4model.com/) for software architecture is an "abstraction-first" approach to diagramming, based upon abstractions that reflect how software architects and developers think about and build software.
+
```

Comment:
Is building these diagrams integrated into CircleCI (and for every build?) or are there separate local install or build steps required to get through diagram generation?

Reply:
These diagrams are not building in CircleCI, they are separate and not even integrated with local build yet for diagram generation.  They are hand-coded and generated either with a [local installation of PlantUML](https://plantuml.com/starting) or using the [PlantUML Online Server](http://www.plantuml.com/plantuml/uml/SyfFKj2rKt3CoKnELR1Io4ZDoSa70000).